### PR TITLE
Simplify tf.keras.backend.bias_add

### PR DESCRIPTION
**Pull request by lgeiger**
`tf.nn.bias_add` now supports `channels_first` and inputs up to rank 5. This means that the implementation of `tf.keras.backend.bias_add` can be greatly simplified by relying directly on `tf.nn.bias_add`.  This should also slightly increase performance since in most cases it can directly rely on `tf.nn.bias_add`.

### CODE REVIEW

**Comment by lgeiger** [comment]
@fchollet Any change you could take a look or reassign the PR?

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -5530,47 +5530,24 @@ def bias_add(x, bias, data_format=None):
...
-        x = x + reshape(bias, (1, bias_shape[0], 1, 1, 1))
-      else:
-        x = x + reshape(bias, (1, bias_shape[3]) + bias_shape[:3])
-    elif data_format == 'channels_last':
-      if len(bias_shape) == 1:
-        x = x + reshape(bias, (1, 1, 1, bias_shape[0]))
-      else:
-        x = x + reshape(bias, (1,) + bias_shape)
-  elif ndim(x) == 4:
+      return x + reshape(bias, (1, bias_shape[3]) + bias_shape[:3])
```
Comment (qlzh727) [inline tensorflow/python/keras/backend.py]: I think this section can be further reduced.

```

if ndim(x) in (3, 4, 5):
  if data_format = 'channels_first':
     bias_reshape_axis = (1, bias_shape[-1]) + bias_shape[:-1]
     return x + reshape(bias, bias_reshape_axis)
  return x + reshape(bias, (1, ) + bias_shape)
```

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Sorry for the long wait.

**Comment by gbaned** [comment]
@lgeiger Could you please check reviewer comments and keep us posted. Thanks!

File: tensorflow/python/keras/backend.py (hunk 1 above)
Comment (lgeiger) [reply tensorflow/python/keras/backend.py]: @qlzh727 Great idea, fixed it in the latest commit.

**Review by qlzh727 — DISMISSED** [review]

**Comment by lgeiger** [comment]
`pylint` wasn't happy about this change, I fixed it in the latest commit. Sorry about that.

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**